From 10cb205223ccbcd9a93bc1af78b8ab421e208b3b Mon Sep 17 00:00:00 2001
From: Geliang Tang <geliang.tang@suse.com>
Date: Fri, 9 Dec 2022 10:24:14 +0800
Subject: [PATCH] Revert "Linux 5.10.158"
Patch-mainline: Never, revert stable version change
References: bsn#83
Modified-by-SEL: No

This reverts commit 592346d5dc9b61e7fb4a3876ec498aa96ee11ac8.
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index f3d1f07b6a6f..bf22df29c4d8 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: GPL-2.0
 VERSION = 5
 PATCHLEVEL = 10
-SUBLEVEL = 158
+SUBLEVEL = 157
 EXTRAVERSION =
 NAME = Dare mighty things
 
-- 
2.35.3

